### PR TITLE
CTCP-1102 Create loading place page

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: test/models/journeyDomain/routeDetails/loading/AdditionalInformationDomainSpec.scala [hunk 1]
```diff
@@ -50,27 +53,27 @@ class AdditionalInformationDomainSpec extends SpecBase with UserAnswersSpecHelpe
 
     "cannot be parsed from UserAnswers" - {
 
-      "when additional contact has no country" ignore {
+      "when additional contact has no country" in {
```
Comment (BledarFeratiEY) [inline test/models/journeyDomain/routeDetails/loading/AdditionalInformationDomainSpec.scala:56]: Should be additional information

File: app/forms/Constants.scala [hunk 2]
```diff
@@ -28,4 +28,5 @@ object Constants {
   lazy val accessCodeLength: Int                            = 4
   lazy val addiationalIdentifierMaxLength: Int              = 4
   lazy val maxLocationOfGoodsAuthorisationNumberLength: Int = 35
+  lazy val locationOfGoodsMaxLength: Int                    = 35
```
Comment (jackheslop96) [inline app/forms/Constants.scala:31]: Might `placeOfLoadingMaxLength` be a more appropriate name?

File: app/pages/routeDetails/loading/PlaceOfLoadingLocationPage.scala [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import controllers.routeDetails.loading.routes
+import models.{Mode, UserAnswers}
+import pages.QuestionPage
+import pages.sections.routeDetails.loading.LocationOfGoodsInformationSection
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case object PlaceOfLoadingLocationPage extends QuestionPage[String] {
+
+  override def path: JsPath = LocationOfGoodsInformationSection.path \ toString
```
Comment (jackheslop96) [inline app/pages/routeDetails/loading/PlaceOfLoadingLocationPage.scala:28]: Wrong section?

File: app/pages/sections/routeDetails/loading/LocationOfGoodsInformationSection.scala [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections.routeDetails.loading
+
+import pages.sections.Section
+import play.api.libs.json.{JsObject, JsPath}
+
+case object LocationOfGoodsInformationSection extends Section[JsObject] {
```
Comment (jackheslop96) [inline app/pages/sections/routeDetails/loading/LocationOfGoodsInformationSection.scala:22]: Rename to `LoadingLocationSection` or similar

File: conf/messages.en [hunk 5]
```diff
@@ -883,6 +881,14 @@ routeDetails.loading.placeOfLoadingAddExtraInformationYesNo.paragraph = This inc
 routeDetails.loading.placeOfLoadingAddExtraInformationYesNo.checkYourAnswersLabel = Do you want to add extra information for the place of loading? //TODO check CYA ticket content
 routeDetails.loading.placeOfLoadingAddExtraInformationYesNo.error.required = Select yes if you want to add extra information for the place of loading
 
+routeDetails.loading.placeOfLoadingLocation.title = Where in {0}} is the place of loading?
```
Comment (jackheslop96) [inline conf/messages.en:884]: Double bracket

File: conf/messages.en [hunk 6]
```diff
@@ -671,9 +671,7 @@ routeDetails.transit.confirmRemoveOfficeOfTransit.heading = Are you sure you wan
 routeDetails.transit.confirmRemoveOfficeOfTransit.error.required = Select yes if you want to remove {0} as an office of transit
 
 routeDetails.transit.checkOfficeOfTransitAnswers.title = Check your answers
-routeDetails.transit.checkOfficeOfTransitAnswers.heading = Check your answers
-
-# Exit
+routeDetails.transit.checkOfficeOfTransitAnswers.heading = Check your answers# Exit
```
Comment (jackheslop96) [inline conf/messages.en:674]: Undo this diff

File: app/pages/sections/routeDetails/loading/LoadingLocationSection.scala [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+package pages.sections.routeDetails.loading
+
+import pages.sections.Section
+import play.api.libs.json.{JsObject, JsPath}
+
+case object LoadingLocationSection extends Section[JsObject] {
+
+  override def path: JsPath = LoadingSection.path \ toString
+
+  override def toString: String = "additionalInformation"
```
Comment (jackheslop96) [inline app/pages/sections/routeDetails/loading/LoadingLocationSection.scala:26]: "location" probably a better js path

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**